docs/public/icons.html

```diff
@@ -0,0 +1 @@
/g></symbol><symbol viewBox="0 0 23 30" id="Test" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M-24-20h70v70h-70z"/><path d="M0 19.953V6.867c0-.746.464-1.421 1.435-1.421.943 0 1.38.675 1.38 1.42v8.838a9.813 9.813 0 011.533-.788V2.49c0-.746.478-1.422 1.407-1.422s1.407.676 1.407 1.422v11.637a13.892 13.892 0 011.534-.17V1.409C8.696.66 9.16 0 10.103 0c.943 0 1.407.661 1.407 1.407v12.65c.563.085 1.084.183 1.534.31V3.25c0-.745.464-1.42 1.407-1.42.957 0 1.407.675 1.407 1.42v13.58c0 1.477.155 2.25.591 2.25.76 0 1.717-3.123 2.505-4.798.718-1.52 1.168-2.068 2.378-2.068.535 0 1.337.535 1.337 1.14 0 .942-.802 2.73-1.224 4.123-.535 1.773-1.464 6.29-4.278 9.273-2.026 2.124-4.56 3.25-7.964 3.25C3.84 30 0 26.89 0 19.953z" fill="currentColor" fill-rule="nonzero"/></g></symbol><symbol viewBox="0 0 21 34" id="Timer1" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M-24-18h70v70h-70z"/><path d="M10.5 0c6.405 0 10 1.586 10 5.25 0 2.922-2.69 6.646-8.03 11.5 5.34 4.854 8.03 8.578 8.03 11.5 0 3.664-3.595 5.25-10 5.25-6.405 0-10-1.586-10-5.25 0-2.922 2.69-6.646 8.03-11.5C3.19 11.896.5 8.172.5 5.25.5 1.586 4.095 0 10.5 0zm0 3l-.67.005c-4.447.08-6.33.97-6.33 2.245 0 1.739 2.207 4.812 6.66 8.922l.34.314.655-.606C15.396 9.916 17.5 6.947 17.5 5.25c0-1.337-2.072-2.25-7-2.25z" fill="currentColor" fill-rule="nonzero"/></g></symbol><symbol viewBox="0 0 32 32" id="Trash1" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M-19-20h70v70h-70z"/><path d="M30.99 6.5l-2.776 17.742c-.44 2.814-.853 3.797-1.513 4.776a6.32 6.32 0 01-2.601 2.225c-1.07.501-2.105.757-4.952.757h-6.296c-2.847 0-3.883-.256-4.952-.757a6.32 6.32 0 01-2.6-2.225c-.66-.98-1.074-1.962-1.514-4.776L1.01 6.5h29.98zm-11.679 5.559l-.114.102L16 15.306l-3.197-3.145-.114-.102a1.526 1.526 0 00-1.902 0l-.115.102-.103.112a1.466 1.466 0 000 1.873l.103.112 3.196 3.147-3.196 3.147a1.467 1.467 0 000 2.098 1.525 1.525 0 002.017.101l.114-.101L16 19.503l3.197 3.147.114.101c.553.444 1.35.444 1.902 0l.115-.101.103-.113c.45-.544.45-1.329 0-1.873l-.103-.112-3.197-3.147 3.197-3.147a1.467 1.467 0 000-2.097 1.525 1.525 0 00-2.017-.102zM27.325.047A4 4 0 0131.288 3.5H.711A4.002 4.002 0 014.675.047h22.65z" fill="currentColor" fill-rule="nonzero"/></g></symbol><symbol viewBox="0 0 36 34" id="Warning1" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M-17-18h70v70h-70z"/><path d="M21.102 1.814L35.35 26.687c1.33 2.322.556 5.3-1.728 6.652a4.728 4.728 0 01-2.409.661H4.787C2.143 34 0 31.822 0 29.135c0-.86.224-1.704.65-2.448L14.897 1.814A3.558 3.558 0 0119.806.496a3.62 3.62 0 011.296 1.318zm-4.897 8.997l.455 10.028c0 1.038.446 1.81 1.34 1.81.894 0 1.34-.772 1.34-1.81l.454-10.028c0-1.038-.803-1.88-1.794-1.88-.992 0-1.795.842-1.795 1.88zM18 30.24c1.321 0 2.393-1.123 2.393-2.507 0-1.385-1.072-2.507-2.393-2.507-1.322 0-2.394 1.122-2.394 2.507 0 1.384 1.072 2.507 2.394 2.507z" fill="currentColor" fill-rule="nonzero"/></g></symbol></svg>
```

Comment:
Why this is a `.html` instead of `.svg`? is because parcel understand HTML code and just reads like a string :D